lib/internal/make-api-call.js

```diff
@@ -103,12 +109,25 @@ exports.inject = function(options) {
       jitter: retryOptions.jitter
     });
 
+    // This function determines whether a response indicates success.
+    // All APIs' JSON responses contain a 'status' field which we use
+    // to determine if the response is successful, apart from the
+    // Geolocation API which uses HTTP POST for its request, and HTTP
+    // status codes for determining success.
+    var isSuccessResponse = function(response) {
+      if (!isPost) {
```

Comment:
I think the reasoning there was that the common case was first (GET), and the rare case (POST) was the second condition, but you're right it does read weirdly, I'll change that.